schema/encoding/jsonschema/benchmark_test.go

```diff
@@ -38,15 +39,25 @@ func BenchmarkEncoder(b *testing.B) {
 			},
 		},
 		{
-			Name:   `Schema=Student`,
+			Name:   `Schema=Simple`,
 			Schema: studentSchema,
 		},
+		{
+			Name:   `Schema=Complex1`,
+			Schema: requestSchema,
+		},
+		{
+			Name:   `Schema=Complex2`,
+			Schema: responseSchema,
+		},
 	}
+	buf := bytes.NewBuffer(make([]byte, 65535))
```

Comment:
This helps, but if any one benchmark was to trigger a Grow (which usually means doubling the buffer size I think), that one benchmark would be unfairly punished. Maybe it's better to just rely on `ioutil.Discard`? Then we don't need to worry about the time spent allocating/growing the buffer at all. Of course, the write itself also becomes free.

Alternatively, we should at the very least move this to inside `for i := range testCases` (before the b.Run call), to assure each benchmarks starts with a writer that has the same capacity.